client/data/deposits/actions.js

```diff
@@ -154,3 +154,61 @@ export function* submitInstantDeposit( transactionIds ) {
 		] );
 	}
 }
+
+export function updateStandardDeposit( data ) {
+	return {
+		type: TYPES.SET_STANDARD_DEPOSIT,
+		data,
+	};
+}
+
+export function* submitStandardDeposit( transactionIds ) {
```

Comment:
This is very similar to `submitInstantDeposit`. This could also be merged into one, but looking at existing actions, having an explicit action for each task seems to be preferred.